piqueserver/networkdict.py

```diff
@@ -7,19 +7,18 @@ def get_network(cidr):
     try:
         return CACHE[cidr]
     except KeyError:
-        network = IPNetwork(cidr)
+        network = ip_network(unicode(cidr))
```

Comment:
This breaks on py3

Reply:
How do I make it compatible with both python 2/3?

Reply:
It requires a unicode object, I guess? There's a few options, but it's not really easy. You could require the callee to give a valid unicode string object, alias unicode to str if unicode is not available or convert it to unicode only if it's a bytes object. We've only had this issue in reverse before.